waspc/data/Generator/templates/server/src/routes/index.js

```diff
@@ -14,6 +14,12 @@ router.get('/', function (req, res, next) {
 
 {=# isAuthEnabled =}
 router.use('/auth', auth)
+
+// NOTE: While we have CORS protection enabled in app.js, it is
+// vitally important that this route always be CORS-protected.
```

Comment:
TODO @shayneczyzewski  - reverify this.

Reply:
Confirmed. Calling `fetch('https://wasp-csrf-demo.herokuapp.com/csrf-token').then(response => response.json()).then(data => console.log(data));` from the console of some random website, you get:
<img width="587" alt="Screenshot 2022-06-21 080124" src="https://user-images.githubusercontent.com/523636/174794405-904a6e83-dcdd-4981-9e0f-a059a4b55ade.png">

Reply:
Although, I'm wondering if I should remove the comment, as I don't think this route being CORS-protected is any more or less important than the others, right?

Reply:
Yeah I can lessen it down a bit :D